frontend/src/app/api/endpointConfigs.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiMethod } from "src/app/api/fetchHelpers";
+import { environment } from "src/constants/environments";
+
+export interface EndpointConfig {
+  basePath: string;
+  version: string;
+  namespace: string;
+  method: ApiMethod;
+}
+
+export const opportunitySearchEndpoint = {
+  basePath: environment.API_URL,
```

Comment:
would like to structure this so basePath could be defaulted, but didn't go that far today